config/routes.rb

```diff
@@ -88,7 +88,7 @@ def matches?(request)
     get "/auth/failure", to: "auth#failure"
 
     resources :claims, only: [:index, :show] do
-      resources :tasks, only: [:index, :show, :create], param: :name, constraints: {name: %r{#{Admin::TasksController::TASKS_SEQUENCE.join("|")}}}
+      resources :tasks, only: [:index, :show, :create], param: :name, constraints: {name: %r{#{ClaimCheckingTasks::TASK_NAMES.join("|")}}}
```

Comment:
Another thought: currently visiting `admin/claims/:claim_id/tasks/student_loan_amount` for a maths and physics claim raises an exception. Should we be returning a 404, same way as we restrict claim journey slugs to those which are relevant to the policy?

Reply:
I don't feel strongly that we need to do this. It feels like an edge case we could ignore because these pages are not public-facing and it would require a service operator to hack the URL to get to the page as it shouldn't be linked to from anywhere.

Reply:
👍